cockroachdb/goose/migration.go

```diff
@@ -0,0 +1,43 @@
+package utils
```

Comment:
issue: package name must match folder name.
```suggestion
package goose
```